css/screens/signupPage.css

```diff
@@ -0,0 +1,33 @@
+/* css for signUp page */
+
+
+.signup-section .signup-container{
+    position: fixed;
+    padding: 2rem;
+    top: 13%;
+    left: 0;
+    right: 0;
+    max-width: 70vw;
+}
+.signup-section .signup-container form{
+    background-color: var(--white-color);
+    border-radius: .5rem;
```

Comment:
You can use px for values less than 1rem.

Reply:
sure...